src/libcore/cell.rs

```diff
@@ -347,6 +349,20 @@ pub enum BorrowState {
     Unused,
 }
 
+/// An error returned by [`RefCell::try_borrow`](struct.RefCell.html#method.try_borrow).
+#[derive(Clone, Debug)]
+#[unstable(feature = "try_borrow", issue = "35070")]
+pub struct BorrowError {
+    _inner: (),
```

Comment:
Yeah, I copied this from the RFC itself, and it can always be removed before being stabilized.